src/js/edu-benefits/components/NavButtons.jsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+
+// TODO update this when progress button moves
+import ProgressButton from '../../common/components/form-elements/ProgressButton';
+
+import { getSectionList } from '../utils/sections';
+
+export default class NavButtons extends React.Component {
+  constructor(props) {
+    super(props);
+    this.sections = getSectionList();
+    this.handleContinue = this.handleContinue.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+  handleContinue(nextPath) {
+    if (this.props.path === '/introduction' || this.props.isValid) {
+      this.props.onNavigate(nextPath);
+      this.props.onComplete(this.props.path);
+    } else if (!this.props.isValid) {
+      this.props.dirtyFields(this.props.path);
+    }
+  }
+  handleSubmit() {
+    if (this.props.isValid) {
+      this.props.onSubmit();
+    }
+  }
+  render() {
+    const { submission, path } = this.props;
+    const currentIndex = this.sections.indexOf(path);
+    const nextPath = currentIndex + 1 < this.sections.length ? this.sections[currentIndex + 1] : null;
+    const previousPath = currentIndex - 1 >= 0 ? this.sections[currentIndex - 1] : null;
+
+    const goBack = () => this.props.onNavigate(previousPath);
+    const goForward = () => this.handleContinue(nextPath);
+
+    const backButton = (
+      <ProgressButton
+          onButtonClick={goBack}
+          buttonText="Back"
+          buttonClass="usa-button-outline"
+          beforeText="«"/>
+    );
+
+    const nextButton = (
+      <ProgressButton
+          onButtonClick={goForward}
+          buttonText="Continue"
+          buttonClass="usa-button-primary"
+          afterText="»"/>
+    );
+
+    if (path === '/review-and-submit') {
+      let submitButton;
+      let submitMessage;
+
+      if (submission.status === false) {
+        submitButton = (
+          <ProgressButton
+              onButtonClick={this.handleSubmit}
+              buttonText="Submit Application"
+              buttonClass="usa-button-primary"/>
+        );
+      } else if (submission.status === 'submitPending') {
+        submitButton = (
+          <ProgressButton
+              onButtonClick={this.handleSubmit}
```

Comment:
Yeah, we probably should. It gets a disabled class, but doesn't appear to actually be disabled.